### PR TITLE
Use javas duration

### PR DESCRIPTION
**Pull request by Waginator**
Removal of HttpClientConfigConstants and ExpirationTime. I've used Javas Duration class instead.

### CODE REVIEW

**Comment by Waginator** [comment]
[Removal of HttpClientConfigConstants and ExpirationTime](https://trello.com/c/kuV6jsnT/9-removal-of-httpclientconfigconstants-and-expirationtime)

File: src/main/java/com/metalr2/config/resttemplate/ApacheHttpClientConfig.java [hunk 1]
```diff
@@ -76,9 +78,9 @@ public void run() {
         // only if connection pool is initialised
         if (pool != null) {
           pool.closeExpiredConnections();
-          pool.closeIdleConnections(IDLE_CONNECTION_WAIT_TIME, TimeUnit.MILLISECONDS);
+          pool.closeIdleConnections(10, MINUTES);
```
Comment (hedgehog1833) [inline src/main/java/com/metalr2/config/resttemplate/ApacheHttpClientConfig.java]: The change from 30 seconds to 10 minutes is on purpose?

File: src/main/java/com/metalr2/config/resttemplate/ApacheHttpClientConfig.java [hunk 2]
```diff
@@ -95,9 +97,9 @@ public TaskScheduler taskScheduler() {
   @Bean
   public CloseableHttpClient httpClient() {
     RequestConfig requestConfig = RequestConfig.custom()
-            .setConnectTimeout(CONNECTION_TIMEOUT)
-            .setConnectionRequestTimeout(REQUEST_TIMEOUT)
-            .setSocketTimeout(SOCKET_TIMEOUT)
+            .setConnectTimeout((int) Duration.ofSeconds(20).toMillis()) // the time for waiting until a connection is established
+            .setConnectionRequestTimeout((int) Duration.ofSeconds(20).toMillis()) // the time for waiting for a connection from connection pool
+            .setSocketTimeout((int) Duration.ofSeconds(30).toMillis()) // the time for waiting for data
```
Comment (hedgehog1833) [inline src/main/java/com/metalr2/config/resttemplate/ApacheHttpClientConfig.java]: Here the times are also changed, but only slightly.

File: src/main/java/com/metalr2/config/resttemplate/ApacheHttpClientConfig.java (hunk 1 above)
Comment (Waginator) [reply src/main/java/com/metalr2/config/resttemplate/ApacheHttpClientConfig.java]: Yipp. We don't need an idle connection cleanup every 30 seconds.

File: src/main/java/com/metalr2/config/resttemplate/ApacheHttpClientConfig.java (hunk 2 above)
Comment (Waginator) [reply src/main/java/com/metalr2/config/resttemplate/ApacheHttpClientConfig.java]: Yeah. I think 20 (or 30) seconds timeout is more than enough.

**Pull request merged**